### ISSUE TITLE
Use of TimeStampedModel raise a RemovedInDjango40Warning 

### ISSUE THREAD

**Issue by jrodriguezva** [jazzband/django-model-utils#450]
## Problem

When I use the class `TimeStampedModel` into my models, there are several RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy() warnings when running test.

[ugettext functions have been deprecated in Django 3.0](https://docs.djangoproject.com/en/dev/releases/3.0/#features-deprecated-in-3-0)

## Environment

- Django Model Utils version: 4.0.0
- Django version: 3.0.3
- Python version: 3.8.5
- Other libraries used, if any: pytest 6.1.1
## Code examples

```python
class Organization(TimeStampedModel):
    name = models.CharField('name', max_length=100)
    email = models.EmailField()
    phone = PhoneNumberField("phone")
    url = models.CharField('url', max_length=50)
```
```bash
=========================================================================================================================== warnings summary ============================================================================================================================
venv/lib/python3.8/site-packages/model_utils/models.py:28
  /adopg-backend/venv/lib/python3.8/site-packages/model_utils/models.py:28: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    created = AutoCreatedField(_('created'))

venv/lib/python3.8/site-packages/model_utils/models.py:29
  /adopg-backend/venv/lib/python3.8/site-packages/model_utils/models.py:29: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    modified = AutoLastModifiedField(_('modified'))

venv/lib/python3.8/site-packages/model_utils/models.py:41
  /adopg-backend/venv/lib/python3.8/site-packages/model_utils/models.py:41: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    start = models.DateTimeField(_('start'), null=True, blank=True)

venv/lib/python3.8/site-packages/model_utils/models.py:42
  /adopg-backend/venv/lib/python3.8/site-packages/model_utils/models.py:42: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    end = models.DateTimeField(_('end'), null=True, blank=True)

venv/lib/python3.8/site-packages/model_utils/models.py:57
  /adopg-backend/venv/lib/python3.8/site-packages/model_utils/models.py:57: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    status = StatusField(_('status'))

venv/lib/python3.8/site-packages/model_utils/models.py:58
  /adopg-backend/venv/lib/python3.8/site-packages/model_utils/models.py:58: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    status_changed = MonitorField(_('status changed'), monitor='status')

-- Docs: https://docs.pytest.org/en/stable/warnings.html
===================================================================================================================== 4 passed, 6 warnings in 0.42s =====================================================================================================================
```

**Comment by hramezani** [comment]
The warning has been satisfied in https://github.com/jazzband/django-model-utils/commit/6fba04ec4ff32b34cca61c6f1770ef6e81a40e2b and we won't see it in the new release of `django-model-utils`.
I created https://github.com/jazzband/django-model-utils/pull/451 to replace `ugettext_lazy` with `gettext_lazy` in `tests/models.py`.

**Comment by smithdc1** [comment]
@hramezani this can now be closed following the release of 4.1.0.

**Comment by hramezani** [comment]
@smithdc1 Yeah, you are right.
